crates/l1_sidecar/src/contracts/mod.rs

```diff
@@ -59,12 +84,11 @@ pub const SUPPORTED_ENCODING_VERSION: u8 = 0;
 ///
 /// Assumes system log verification and DA input verification are disabled.
 pub fn commit_batches_shared_bridge_call(
-    l2_chain_id: L2ChainId,
     last_committed_l1_batch: &L1BatchWithMetadata,
     batch: &L1BatchWithMetadata,
 ) -> impl SolCall {
     IExecutor::commitBatchesSharedBridgeCall::new((
-        alloy::primitives::U256::from(l2_chain_id.as_u64()),
+        alloy::primitives::Address::ZERO, // This value is not currently used in the implementation
```

Comment:
But still needed for V28, right? Could be another reason why some test are failing.
On this note - support for V26/V27 can be dropped at this point if that makes your life easier.

Note: multiple places in this file with the same logic, above applies to all of them